public/components/Flyout/flyouts/components/AlertsDashboardFlyoutComponent.js

```diff
@@ -153,7 +167,7 @@ export default class AlertsDashboardFlyoutComponent extends Component {
   };
 
   getAlerts = async () => {
-    this.setState({ ...this.state, loading: true });
+    this.setState({ loading: true });
```

Comment:
Why are we removing `...this.state`?

Reply:
I don't believe that bit is necessary. My understanding is that the `setState` method will update the `state` with only the key/value pairs provided in its argument. So if the `state` contains an `alerts` list and a `loading` boolean, and `setState` should only update the `loading` value when executing `this.setState({ loading: true })`.